src/routes/about/+page.svelte

```diff
@@ -85,28 +85,34 @@
 				the low-hanging fruit like ephemeral 1-to-1 chats, but Felt is designed to be a powerful programmable
 				platform for communities, which makes privacy and security with good UX an extremely challenging
 				problem. Matrix prioritizes privacy and security in a federated context; we prioritize UX in
-				a centralized context. What goes for Mastodon goes for Felt: don't send anything in DMs that
-				you don't trust the admin with, or anything sensitive at all.
+				a centralized context. What goes for Mastodon and similar platforms goes for Felt: don't send
+				sensitive information that you can't trust your admin to keep safe.
 			</p>
 		</details>
 		<details>
 			<summary> Why isn't Felt decentralized or federated? </summary>
 			<p>
-				We expect our tech — which is focused on <em>communities</em>, not <em>messaging</em> — to eventually
-				support a decentralized protocol standard, perhaps with ActivityPub or Matrix or both, but today
-				we are focused on UX quality and implementation velocity. We find it much easier to deliver good
-				UX with a centralized architecture given the nature of decentralized systems and our small self-funded
-				team with no expertise in such things. If Matrix fits your usecases, we recommend considering
-				using it instead of Felt. For microblogging, we recommend Mastodon.
+				We expect our tech — which is focused on <em>communities</em>, not <em>messaging</em> — to
+				eventually support a decentralized protocol standard, perhaps with ActivityPub or Matrix or
+				both, but today we are focused on UX quality and implementation velocity. It's much easier
+				for our small self-funded team to deliver good UX with a centralized architecture given the
+				nature of decentralized systems, and federation is less important for the small communities
+				we're designing for. If Matrix fits your usecases, we recommend considering using it instead
+				of Felt. For microblogging and other
+				<a href="https://publicinfrastructure.org/2023/03/23/new_public-a-social-network-taxonomy/"
+					>big room social media</a
+				>, we recommend Mastodon and the Fediverse.
 			</p>
 			<p>
 				Felt's focus is on communities and their custom needs, with an emphasis on delivering a
 				polished UX in smaller scale, higher trust environments. We are however attempting to follow
-				the standard ActivityPub/Fediverse/Mastodon/etc
+				the standard ActivityPub/Fediverse/Mastodon
```

Comment:
```suggestion
				the standard
```
This seems kind of verbose